src/components/change-requests/new-change-request/new-change-request-page/common-form-fields/common-form-fields.tsx

```diff
@@ -4,39 +4,60 @@
  */
 
 import { Form } from 'react-bootstrap';
-import { exampleAllWorkPackages } from '../../../../../test-support/test-data/work-packages.stub';
-import { exampleAllProjects } from '../../../../../test-support/test-data/projects.stub';
 import { wbsPipe } from '../../../../../shared/pipes';
+import { Project, WorkPackage } from 'utils';
 import './common-form-fields.module.css';
+import { CR_Type } from '@prisma/client';
+
+interface IProp {
+  setFormType: React.Dispatch<React.SetStateAction<CR_Type>>,
+  projects: Project[],
+  workPkgs: WorkPackage[],
+  handleChange: (e: any) => void,
+}
+
+const crTypeDisplayName = {
```

Comment:
we actually don't want this. the enum names are the correct ones, and display as a all-caps snake case string is fine for MVP